### PR TITLE
Add csv_excel option

### PR DESCRIPTION
**Pull request by ausi**
See https://github.com/contao/contao/pull/1870

### CODE REVIEW

File: docs/manual/form-generator/forms.de.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ immer als Attachment angehängt.
 | Rohdaten                 | Die E-Mail enthält die unbearbeiteten Daten, das heißt, die Inhalte der einzelnen Formularfelder werden einfach untereinander aufgelistet. |
 | XML-Datei                | Der E-Mail ist eine XML-Datei mit den Formulardaten angehängt. |
 | CSV-Datei                | Der E-Mail ist eine CSV-Datei mit den Formulardaten angehängt. |
+| CSV-Datei (Microsoft Excel) | Der E-Mail ist eine CSV-Datei im Microsoft-Excel-Format mit den Formulardaten angehängt. |
```
Comment (ausi) [inline docs/manual/form-generator/forms.de.md]: This is 4.10 only. Should I write this directly into the text or is there a snippet for that?

Comment (Toflar) [reply docs/manual/form-generator/forms.de.md]: There's a snippet :) `{{< version "4.10" >}}`

Comment (ausi) [reply docs/manual/form-generator/forms.de.md]: Added in d459a3836123587361269c4205532259e3f14a19

**Comment by netzarbeiter** [comment]
Thank you, @ausi.

**Pull request merged**